foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigMapping.java

```diff
@@ -17,20 +17,43 @@
 
 package org.apache.servicecomb.config;
 
-import java.io.InputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.HashMap;
 import java.util.LinkedHashMap;
+import java.util.List;
 import java.util.Map;
 
+import org.apache.commons.configuration.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Created by   on 2017/1/5.
  */
 public final class ConfigMapping {
   private static Map<String, Object> configMap = null;
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigMapping.class);
+
   static {
     ClassLoader loader = Thread.currentThread().getContextClassLoader();
-    InputStream is = loader.getResourceAsStream("mapping.yaml");
-    configMap = YAMLUtil.yaml2Properties(is);
+    List<URL> urlList = new ArrayList<>();
+    configMap = new HashMap<String, Object>();
+    Enumeration<URL> urls;
+    try {
+      urls = loader.getResources("mapping.yaml");
+      while (urls.hasMoreElements()) {
+        urlList.add(urls.nextElement());
+      }
+      for (URL url : urlList) {
+        configMap.putAll(YAMLUtil.yaml2Properties(url.openStream()));
```

Comment:
From YAMLUtil.yaml2Properties docs, it's not mentioned if close the stream. Close the stream if necessary or add a comment that will close internally.

Reply:
snake yaml's demo and document don't require to close the stream. but we close the stream maybe more safely

Reply:
"snake yaml's demo and document don't require to close the stream"
it's not a good idea to work in this way.

1.YAMLUtil.yaml2Properties is our code
2.if you are not sure if "yaml.loadAll" will close stream, it's better to read the source code, not guess

Reply:
yes, i have read the source, i don't find no code to close the stream, but their demo code do not close the stream. Today i write a demo, if i do not close the stream , the stream will be leak.
![image](https://user-images.githubusercontent.com/1768102/41083407-ba14da58-6a63-11e8-941d-021a804b21c4.png)
so add an api document to explain this.